flake.nix

```diff
@@ -103,6 +101,33 @@
             ];
           };
 
+          vault = pkgs.buildGoModule rec {
+            pname = "vault";
+            version = "1.14.8";
```

Comment:
any specific reason why we are building vault package here?

Reply:
Forgot to add it to the services, not it works:

<img width="1898" alt="image" src="https://github.com/bank-vaults/vault-operator/assets/113284287/f0d15de3-46b0-462d-a159-4fd0f6709a77">